CONTRIBUTING.md

```diff
@@ -44,52 +50,45 @@ Before running this command make sure Docker is running. Read more about [tests]
 
 ## Developing
 
-### Adding a component
+### Directory structure
 
-We welcome UI components contributions! If you'd like to get your component added to iTwinUI, follow these guidelines:
+- **`src/` contains all the .scss files**.
+  - After running build, the generated .css files are put in `lib/css/` folder (in project root).
+- **`backstop/tests/` contains the .html files** that uses those built styles.
+  - After running build, the minified .html files are put in `backstop/minified/` folder.
+- **`backstop/scenarios/` contains the .js files** where visual test scenarios are defined.
 
-- Make a new folder with the name of the component under `src` folder.
-- Break variables and mixins into separate files where possible.
-- Make sure to include a `classes.scss` file in the folder as this file is used to generate all the relevant css classes.
+### Adding a new component
 
-  - This file should simply define classes using your mixins.
-  - Example from `src/toggle-switch/classes.scss`, where index imports all mixins and relevant scss.
-
-    ```scss
-    @import './index';
-
-    .iui-toggle-switch {
-      @include iui-toggle-switch;
-    }
-    ```
+If you'd like to get your component added to iTwinUI, follow these guidelines.
```

Comment:
I feel here should not be used 'would'... 
![image](https://user-images.githubusercontent.com/81580355/119096171-e8f22a00-ba1b-11eb-9d05-34fd6d7f0550.png)

Reply:
what 😄